.circleci/config.yml

```
@@ -262,7 +270,29 @@ jobs:
       - slack/notify:
           color: '#1d990c'
           message: '${CIRCLE_USERNAME} deployed *${BUILD_TAG}* to *Development*'
-          webhook: ${SLACK_WEBHOOK_CT}
+          webhook: ${SLACK_WEBHOOK}
+
+jobs:
+
+  build_and_test:
+    <<: *test_container_config
+    <<: *defaults
+
```

Add this back in to speed up the testing section
```suggestion
    resource_class: xlarge
    
```